test/integration/auth-methods/digest.test.js

```diff
@@ -1,7 +1,25 @@
-var expect = require('chai').expect;
+var expect = require('chai').expect,
+    {createDigestServer} = require('../../fixtures/server');
 
 describe('digest auth', function () {
-    var testrun;
+    var PORT = 2000,
+        USERNAME = 'postman',
+        PASSWORD = 'password',
+        digestServerURL = 'http://localhost:' + PORT,
+        digestServer,
+        testrun;
+
+    before(function () {
+        digestServer = createDigestServer({
+            username: USERNAME,
+            password: PASSWORD,
+            PORT
+        });
+    });
+
+    after(function () {
+        digestServer.destroy();
+    });
 
     // @todo
```

Comment:
check if possible now

Reply:
still not possible, as this server also return qop-value